new-frontiers-14/frontier-station-14
# Inheritable Vessel and POI Prototypes

## About the PR
Makes VesselPrototype and PointOfInterestPrototype inheritable.

Pushes commonly defined items in Vessel and POI prototypes into base parents.

The NFSD outpost is a slightly lighter brown for map readability, the cook role's been removed from the Empress, and engine info for the Tyne and Paladin were added.

## Why / Balance
#2631 wouldn't have been needed with this.

## How to test
Check Tinnia's, make sure the bus stops there (grid has StationTransitComponent) and it can't be moved (linear & angular damping in ShuttleComponent set to 999999).

Check LPBravo, make sure it's hidden.

Purchase regular ships, they should have their IFF label on and appear white.

Purchase pirate ships, they should have their IFF label off and still appear white.

## Media
N/A

## Requirements



## Breaking changes
Existing shuttle PRs could use an update.

**Changelog**
Nothing player facing, no changelog.
